api/DocumentOrShadowRoot.json

```diff
@@ -467,106 +467,6 @@
           }
         }
       },
-      "nodeFromPoint": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/nodeFromPoint",
-          "support": {
-            "chrome": {
-              "version_added": false
-            },
-            "chrome_android": {
-              "version_added": false
-            },
-            "edge": {
-              "version_added": false
-            },
-            "firefox": {
-              "version_added": false,
-              "notes": "Implemented in Firefox but currently only works in chrome code."
```

Comment:
https://github.com/mozilla/gecko-dev/blob/f98dad153b59a985efd4505912588d4651033395/dom/webidl/DocumentOrShadowRoot.webidl#L18-L23 confirms this. There's a reference to https://github.com/w3c/csswg-drafts/issues/556 which is still open, there isn't anything to implement yet.